packages/terra-site/src/examples/content-container/ContentContainerFill.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 import ContentContainer from 'terra-content-container';
-import Button from 'terra-button';
+import Heading from 'terra-heading';
 
 const display1 = <p key="1">display1display1display1display1display1display1display1display1display1display1display1display1display1display1</p>;
 const display2 = <p key="2">display2display2display2display2display2display2display2display2display2display2display2display2display2display2</p>;
```

Comment:
This doesn't necessarily need to be done on this pull request. But at some point we should update this display text to provide a better example. 

A quick update could be using lipsum text: http://www.lipsum.com/

SD / UX might be able to provide a better ideal template. Perhaps a header with a list or other more realistic header / content examples.

Reply:
Here are a couple examples I've thrown together with the ContentContainer. This view is composed of 3 ContentContainers.

![image](https://user-images.githubusercontent.com/6720991/31355767-2bd687de-ad01-11e7-8a16-79c34a52d0ed.png)

Reply:
Updated display text using lipsum text for now. https://github.com/cerner/terra-core/commit/5e3d8f5b899dd44e291a630c354074c4c4c85a76